### PR TITLE
Added additional options for HELMET

### PR DESCRIPTION
**Pull request by medjedovicm**
# Implementation

- Added .env option `HELMET_CSP_CONFIG_PATH` that will import the JSON config to HELMET `contentSecurityPolicy` directives
- Added .env option `HELMET_COEP` that will be bassed to the HELMET config for `crossOriginEmbedderPolicy`

### CODE REVIEW

**Comment by allanbowe** [comment]
Per discussion, let's switch this to using a defined set of CSP directives and disable the "disable" option

File: web/package.json [hunk 1]
```diff
@@ -20,6 +20,7 @@
     "@types/node": "^12.20.28",
     "@types/react": "^17.0.27",
     "axios": "^0.24.0",
+    "monaco-editor": "^0.33.0",
```
[the remark this answers is not in the document]
Comment (allanbowe) [reply web/package.json:23]: @medjedovicm - not sure this explains it, as the build runs in a pipeline.  Does the built version work without it?

File: api/src/utils/parseHelmetConfig.ts [hunk 2]
```diff
@@ -0,0 +1,33 @@
+import path from 'path'
+import fs from 'fs'
+
+export const getEnvCSPDirectives = (
+  HELMET_CSP_CONFIG_PATH: string | undefined
+) => {
+  let cspConfigJson = {
+    'script-src': ["'self'", "'unsafe-inline'"]
```
[the remark this answers is not in the document]
Comment (medjedovicm) [reply api/src/utils/parseHelmetConfig.ts:8]: @saadjutt01 override helmet default with this. So I just kept it that way. The file if included will override both.

File: web/package.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (saadjutt01) [reply web/package.json:23]: I see, new package `react-monaco-editor`, has `monaco-editor` as peerDependency.

**Review by saadjutt01 — APPROVED** [review]

**Pull request merged**